app/src/organisms/ModuleCard/ModuleOverflowMenu.tsx

```diff
@@ -52,13 +78,13 @@ export const ModuleOverflowMenu = (
             ] as MenuItemsByModuleType[ModuleType]).map(
               (item: any, index: number) => {
                 return (
-                  <>
+                  <React.Fragment key={`${index}_${module.moduleType}`}>
```

Comment:
just my preference 😂 so completely optional

```suggestion
                  <React.Fragment key={`${module.moduleType}_${index}`}>
```